mitodl/hugo-course-publisher
# video / pdf page desktop header fix

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
Fixes a bug where the header does not display properly on video and pdf pages

#### How should this be manually tested?
Run the site and visit the deploy preview, ensure that the header is being rendered properly.  Here are a few examples:

http://localhost:3000/courses/ec-711-d-lab-energy-spring-2011/sections/intro-energy-basics-human-power/mitec_711s11_read_react/
http://localhost:3000/courses/ec-711-d-lab-energy-spring-2011/sections/week-7-trip-planning-and-preparations/lecture-7-solar-cookers-creative-capacity-building-trip-preparation/

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/99708753-3b9e5980-2a6c-11eb-8a18-e2291eb62bab.png)
